build.sbt

```diff
@@ -16,16 +19,16 @@ assemblyOption in assembly ~= {
   _.copy(cacheUnzip = true)
 }
 
-val pureconfigVersion = "0.8.0"
+val pureconfigVersion = "0.12.1" //didn't work with 0.11.1 but works with this strangely
 
 //Define dependencies.
 libraryDependencies ++= Seq(
-  "org.scalatest"         %% "scalatest"             % "3.0.1" % "test",
-  "org.scalacheck"        %% "scalacheck"            % "1.13.4" % "test",
+  "org.scalatest"         %% "scalatest"             % "3.2.0-M1" % "test", //"3.0.1" % "test",
```

Comment:
Remove commented code. Is there a non-milestone version available?

Reply:
Yes there is. I will test with the non-milestone version and recommit modified code